clear-containers.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Clear Containers Agent
+Documentation=https://github.com/clearcontainers/agent
+
+[Service]
+ExecStart=@bindir@/@ccagent@
```

Comment:
@sboeuf it would be great not to restart.

In our existing agent we have https://github.com/01org/cc-oci-runtime/blob/master/data/cc-agent.service#L9
FailureAction=poweroff

This makes debugging certain set of failures very hard if not impossible. But I do not know a better way to handle it, so that we can turn this off selectively.